compute_endpoint/tests/unit/test_bad_endpoint_config.py

```diff
@@ -1,7 +1,5 @@
 import pytest
-from globus_compute_endpoint.executors.high_throughput.executor import (
-    HighThroughputExecutor,
-)
+from globus_compute_endpoint.executors import HighThroughputExecutor
```

Comment:
Should this be testing against `HighThroughputEngine`, now?

Reply:
Let me fix that.